packages/control-property-editor/src/slice.ts

```diff
@@ -133,7 +133,30 @@ export const initialState: SliceState = {
     },
     dialogMessage: undefined,
     appMode: 'adaptation',
-
+    changeStack: {
+        canUndo: false,
+        canRedo: false
+    },
+    canSave: false,
+    isAppLoading: true
+};
+export const adpState: SliceState = {
```

Comment:
This code is only used for tests and it should not be here. Please remove it.

Reply:
Done in [6c93187](https://github.com/SAP/open-ux-tools/pull/2068/commits/6c9318770081a6025469af63e0a5939a2a2ef178)